models/ios/logging_global/merged_example_01.txt

```diff
@@ -0,0 +1,64 @@
+# Using merged
```

Comment:
The cli_rm_builder doesn't pick up examples. These need to be directly added to the module file. Please be aware of an existing issue though - https://github.com/ansible-network/cli_rm_builder/issues/2.

Reply:
okay, so I won't put it in the module as of now, Thanks @NilashishC